release-items.md

```diff
@@ -13,16 +13,16 @@
 
 ## updating the website
 
-For 5.x
+For 6.x
 
 0. Change to the master branch
-1. execute `make docs` (when this process completes you'll be on the gh-pages branch)
-2. `git commit -a -m 'chore: website 5.x.x'`
+1. execute `docs:prepare:publish:stable` (when this process completes you'll be on the gh-pages branch)
+2. `git commit -a -m 'chore: website 6.x.x'`
 3. `git push origin gh-pages`
 
-For 4.x
+For 5.x
 
-0. Change to the 4.x branch
+0. Change to the 5.x branch
 1. execute `make docs_legacy` (when this process completes you'll be on the gh-pages branch)
```

Comment:
is `make docs_legacy` still available now that we deleted _Makefile_?

Reply:
I think v5.x branch has still the makefile. It would need another PR for the 5.x branch to make it

Reply:
Oh, missed that 👍